ms2/src/org/labkey/ms2/proteinSequence.jsp

```diff
@@ -38,4 +38,4 @@
         </tr>
     </table>
 <% } %>
-<big><tt><%= bean.protein.getFormattedSequence(bean.run) %></tt></big>
+<big><tt><%= h(bean.protein.getFormattedSequence(bean.run)) %></tt></big>

```

Comment:
Rosaline had this change as well. Look at getFormattedSequence()... it generates HTML (see the tag constants). She already changed that method to return HtmlString, so you can revert this change.